pygyw/layout/drawings.py

```diff
@@ -199,8 +195,8 @@ def _line_to_commands(self, line: str, top: int) -> "list[commands.BTCommand]":
         ctrl_data = bytearray([commands.ControlCodes.DISPLAY_TEXT])
         ctrl_data += self.left.to_bytes(2, 'little', signed=True)
         ctrl_data += top.to_bytes(2, 'little', signed=True)
-        ctrl_data += bytes(self.font.prefix if self.font is not None else "NUL", 'utf-8')
-        ctrl_data += (self.size if self.size is not None else 0).to_bytes(1, 'little')
+        ctrl_data += bytes(self.font.filename.ljust(5, '\0'), 'utf-8')
```

Comment:
Like with the Flutter library, can you add an assert on the filename length instead ?